src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationOptions.cs

```diff
@@ -110,6 +125,10 @@ public string Caption
         ///     <see cref="System.Security.Claims.ClaimsIdentity" />.
         /// </summary>
         public string SignInAsAuthenticationType { get; set; }
+        /// <summary>
+        ///     Gets or sets the display—(Optional) 
+        /// </summary>
+        public Display DisplayMode { get; set; }
```

Comment:
This should default to page

Reply:
Display is an enum with Page as the default value - I can make it explicit if you would prefer?

Reply:
No its fine thanks

Reply:
good to complete PR then?